package.json

```diff
@@ -81,7 +81,7 @@
         "rollup-plugin-istanbul": "^2.0.1",
         "rollup-plugin-json": "^4.0.0",
         "rollup-plugin-node-resolve": "^5.2.0",
-        "select2": "3.5.1",
+        "select2": "^4.1.0-rc.0",
```

Comment:
Have you checked what breaking changes are announced in `select2` v4? Could it affect us?

Reply:
**issue:** This breaks TAO. We cannot upgrade Select2 for the time being, the code is not compatible, and a significant migration is required. Moreover, we should not use RC without a very good reason.